python/ray/serve/benchmarks/microbenchmark.py

```diff
@@ -61,55 +59,32 @@ async def do_queries(self, num, data):
             await fetch(self.session, data)
 
 
-async def trial(
-    result_json,
-    intermediate_handles,
-    num_replicas,
-    max_batch_size,
-    max_concurrent_queries,
-    data_size,
+def build_app(
+    sync_callable: bool,
+    intermediate_handles: bool,
+    num_replicas: int,
+    max_batch_size: int,
+    max_concurrent_queries: int,
 ):
-    trial_key_base = (
-        f"replica:{num_replicas}/batch_size:{max_batch_size}/"
-        f"concurrent_queries:{max_concurrent_queries}/"
-        f"data_size:{data_size}/intermediate_handle:{intermediate_handles}"
-    )
-
-    logger.info(
-        f"intermediate_handles={intermediate_handles},"
-        f"num_replicas={num_replicas},"
-        f"max_batch_size={max_batch_size},"
-        f"max_concurrent_queries={max_concurrent_queries},"
-        f"data_size={data_size}"
+    @serve.deployment(
+        num_replicas=num_replicas,
+        max_concurrent_queries=max_concurrent_queries,
     )
+    class SyncCallee:
+        def __init__(self):
+            logging.getLogger("ray.serve").setLevel(logging.ERROR)
 
-    deployment_name = "api"
-    if intermediate_handles:
-        deployment_name = "downstream"
-
-        @serve.deployment(name="api", max_concurrent_queries=1000)
-        class ForwardActor:
-            def __init__(self):
-                self.handle = None
-
-            async def __call__(self, req):
-                if self.handle is None:
-                    self.handle = serve.get_deployment(deployment_name).get_handle(
-                        sync=False
-                    )
-                obj_ref = await self.handle.remote(req)
-                return await obj_ref
-
-        ForwardActor.deploy()
-        routes = requests.get("http://localhost:8000/-/routes").json()
-        assert "/api" in routes, routes
+        def __call__(self, req):
+            return b"ok"
 
     @serve.deployment(
-        name=deployment_name,
         num_replicas=num_replicas,
         max_concurrent_queries=max_concurrent_queries,
     )
-    class D:
+    class AsyncCallee:
```

Comment:
What's the difference b/w Sync and Async targets here (besides batching)?

Should we consider making different targets mimicking diff workloads? For ex, CPU-bound, blocking I/O, etc